pom.xml

```diff
@@ -57,7 +57,7 @@
         <spring.security.version>4.0.3.RELEASE</spring.security.version>
```

Comment:
You can update all dependencies to their latest versions and especially to Java 8.

You must take a look at the pac4j pom.xml file and copy the pmd and findbugs plugin.